tests/unit/dash/development/test_base_component.py

```diff
@@ -540,15 +540,19 @@ def assert_no_trailing_spaces(self, s):
         for line in s.split('\n'):
             self.assertEqual(line, line.rstrip())
 
+    def match_lines(self, val, expected):
+        for val1, exp1 in zip(val.splitlines(), expected.splitlines()):
+            assert val1 == exp1
```

Comment:
line-by-line matching, just to help me figure out where exactly the differences were